src/controllers/validators/manageroles.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require('@hapi/joi');
+const { permissions } = require('../../models/schema/permissions');
+
+const CreateRole = Joi.object().keys({
+    name: Joi.string().required(),
+    rolePermissions: Joi.array().items(Joi.string().valid(...permissions)),
```

Comment:
Would be good to rename `rolePermissions` into `permissions`.
Import `permissions` list from schema as some other name.
```javascript
const { permissions: listedPermissions } = require('../../models/schema/permissions');
```
Also `permissions` should be a required field. If a GET request is sent without permissions the service would throw an error:

```json
{
    "error": {},
    "message": "Cannot read property 'map' of undefined"
}
```

Also, **SUGGESTION** should the name of the role be unique. Otherwise, it could be confusing to the user.